bib/cmd/bootc-image-builder/image.go

```diff
@@ -42,6 +42,9 @@ type ManifestConfig struct {
 
 	// TLSVerify specifies whether HTTPS and a valid TLS certificate are required
 	TLSVerify bool
+
+	// Use a local container from the host rather than a repository
+	Local bool
```

Comment:
Was it discussed why not to just *always* fetch the image to containers-storage as a preparatory step, hence squashing the distinction between "remote" and "local"?

Reply:
It was discussed, yes. But I don't see how this is not needed even if we copy a remote container into `containers-storage`